jquery.dynatable.js

```diff
@@ -315,6 +315,11 @@
 
       td += '"';
     }
+    
+    // allow cells to be editable
+    if (column.contentEditable) {
+      td += ' contenteditable="true"';
```

Comment:
The `td` is created as a string, so this is the only way to append the attribute.

That's true. One approach is to use a custom class `editable`/`columnEditable` as a check instead?